src/VaultV2.sol

```diff
@@ -318,12 +317,12 @@ contract VaultV2 is IVaultV2 {
     /* TIMELOCKS FOR CURATOR FUNCTIONS */
 
     function submit(bytes calldata data) external {
-        bytes4 selector = bytes4(data);
         require(msg.sender == curator, ErrorsLib.Unauthorized());
         require(executableAt[data] == 0, ErrorsLib.DataAlreadyPending());
-        require(timelock[selector] != type(uint256).max, ErrorsLib.Abdicated());
 
-        executableAt[data] = block.timestamp + timelock[selector];
+        bytes4 selector = bytes4(data);
+        bytes4 timelockSelector = selector == IVaultV2.decreaseTimelock.selector ? bytes4(data[4:8]) : selector;
+        executableAt[data] = block.timestamp + timelock[timelockSelector];
```

Comment:
otherwise a function? Could be useful for integrator? Not convinced 
```suggestion
        executableAt[data] = block.timestamp + timelock(data);
```